Makefile

```diff
@@ -23,6 +23,9 @@ list:
 	$(DEPS)
 	$(CDK) list
 
+run-test:
```

Comment:
Might be easier to follow if you start with test/stacks.test.ts: https://github.com/aws-quickstart/ssp-amazon-eks/pull/184/files#diff-59f0283006bf485e281b97535ff10c05afe6253b30c4ab3b6b298851ffd3579b

Reply:
tests in this project are a wonderful thing. Thank you.